docs/tutorials/01_neural_networks.ipynb

```diff
@@ -7,16 +7,52 @@
    "source": [
     "# Quantum Neural Networks\n",
     "\n",
-    "This notebook demonstrates the different generic quantum neural network (QNN) implementations provided in Qiskit Machine Learning.\n",
-    "The networks are meant as application-agnostic computational units that can be used for many different use cases. \n",
-    "Depending on the application, a particular type of network might be more or less suitable and might require to be set up in a particular way.\n",
-    "The following different available neural networks will now be discussed in more detail:\n",
+    "## Overview\n",
+    "This notebook demonstrates the different generic `quantum neural network (QNN)` implementations provided in Qiskit Machine Learning.\n",
     "\n",
-    "1. `NeuralNetwork`: The interface for neural networks. This is an abstract class.\n",
+    "The Tutorial is structured as follows:\n",
+    "1. [Introduction](#1.-Introduction)\n",
+    "2. [NeuralNetwork](#2.-NeuralNetwork)\n",
+    "3. [EstimatorQNN](#3.-EstimatorQNN)\n",
+    "    * Defining the quantum circuit and the observable\n",
+    "    * Constructing EstimatorQNN using the circuit and observable\n",
+    "    * Specifying input and weights\n",
+    "    * Forward and Batched Forward Pass\n",
+    "    * Backward and Batched Backward Pass\n",
+    "    * Constructing a more complex EstimatorQNN\n",
+    "4. [SamplerQNN](#4.-SamplerQNN)\n",
+    "    * Defining a quantum circuit\n",
+    "    * Constructing SamplerQNN using the circuit\n",
+    "    * Specifying input and weights\n",
+    "    * Forward Pass and Backward Pass\n",
```

Comment:
```suggestion
    "4. [SamplerQNN](#4.-SamplerQNN)\n",
```

Reply:
I think that in this case the index ends up being too long if we list sub-sections, so we can keep it high-level.